README.permut

```diff
@@ -1,7 +1,7 @@
 The package `permut' contains some functions to deal with
 permutability in finite groups.
 
-To install this package, just decompress the file permut-1.03.tar.bz2
+To install this package, just decompress the file permut-1.04.tar.bz2
```

Comment:
I would actually consider writing something like `permut-X.Y.tar.bz2` to avoid editing this file manually each time the new release is made.